src/modules/navigator/rtl_params.c

```diff
@@ -107,32 +107,32 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
 /**
  * Return type
  *
- * Fly straight to the home location or planned mission landing and land there or
+ * Fly straight to the return location or planned mission landing and land there or
  * use the planned mission to get to those points.
  *
  * @value 0 Return home via direct path
  * @value 1 Return to a planned mission landing, if available, via direct path, else return to home via direct path
  * @value 2 Return to a planned mission landing, if available, using the mission path, else return to home via the reverse mission path
- * @value 3 Return via direct way to whatever is closest: home, mission lading or safe point
+ * @value 3 Return via direct path to closest destination: home, mission landing pattern or safe point
```

Comment:
@sfuhrer Values 0, 1, 2 state that the default is back to home if no other path can be found. Is that still correct? [It should be for 2, but arguably 1 might go to a rally point]

Reply:
With #12696 nothing changed for 0,1,2. Safe landing points are thus only considered for 3. That's how it's implemented now, but I'm open for other ideas!

Your rephrasing makes sense.

Reply:
Thanks @sfuhrer I don't have merge rights on this repo, so feel free to merge too :-). That is just what I needed to know. 

> but I'm open for other ideas!

I don't see any reason **not** to use the rally points as a fallback for all cases (though they won't likely be used with option 2, since a reverse mission will take you close to home). If I want to emulate current behaviour I just don't define/clear my rally points. 

Or to put it another way, if I decided to have rally points defined, why wouldn't I want to use those rather than home?

Reply:
@hamishwillee my neither, but I will raise it:) 

About the why not using Rally Points always: From my perspective (heavily VTOL sided), mission landings are much safer than landings at Home or Rally points, so I want the user to be sure that he really needs a landing location at Home/Rally Point. Also, with how it is currently implemented, with option 3 we also always take both Home and Mission landing into account (as I think this is necessary for big missions (potentially BVLOS). So I think we anyway would need another RTL type (additionally to the 3 there were prior this PR). And as from my perspective the need for choosing between Home and Landing is anyway coupled with the need for Rally Points, I ended up enabling them only in type 3. 
Does that make sense?

Reply:
@sfuhrer No, it really does not make sense :-).  You want to obey the mission landing if defined, but the _fallback_ can and should be either home or rally point in all cases.

For type 0 IMO you use rally point or home. I don't see any case where you'd define rally points and then NOT use them in preference to home. 

For type 1 it doesn't really matter because mission upload checks would prevent you uploading a mission without a landing pattern if RTL_TYPE=1. BUT if it was allowed, then what I meant was "use rally points and home" as _fallback_  (not just home). Where is the downside.

For type 2 the same argument holds - as a fallback there is little difference between home and rally points.

Type 3 is odd. The only case you're using rally points currently, but you're ignoring the defined mission landing. 

I hope that makes sense :-). If not, then ping me on slack and we can chat it through.